packages/ui/components/apps/AppCard.tsx

```diff
@@ -75,6 +81,14 @@ export function AppCard({ app, credentials, searchText }: AppCardProps) {
         {app.description}
       </p>
 
+      {app.dependencyData && !app.dependencyData?.installed && (
+        <Link
+          className="mt-5 rounded-md bg-blue-100 p-0.5 text-sm font-semibold text-blue-900 underline"
+          href={`${CAL_URL}/apps/${app.dependency}`}>
+          {t("requires_app", { dependencyName: app.dependencyData.name })}
+        </Link>
+      )}
```

Comment:
Display this on the app card if the dependency is not installed. @Jaibles thoughts?

![image](https://user-images.githubusercontent.com/65426560/221754161-411572d7-97a3-4931-9610-5d0b4c34c250.png)

Reply:
If Typeform requires Routing Form, should we move Typeform to EE? I have a feeling it might cause confusion if someone wants to install Typeform.

Reply:
Moving to ee is not required now. In fact we have to move routing forms out of ee. 
License component tells the user if it's enterprise or not.